kubemarine/procedures/check_iaas.py

```diff
@@ -257,7 +258,10 @@ def system_distributive(cluster):
                 cluster.log.error('Host %s running unknown OS family \"%s\"' % (address, detected_os))
             else:
                 detected_supported_os.append(detected_os)
-                cluster.log.debug('Host %s running \"%s\"' % (address, detected_os))
+                cluster.log.debug('Host %s running \"%s\"' % (address, detected_os)) 
+                detected_versions_compatability.append(detected_os)
```

Comment:
`detected_os` is made from os.name and os.version. But in fact os.name can be different but os.family be the same. In Kubemarine, we consider nodes with the same os.family and os.version effectively the same. os.name has only informational sense. I suggest to postpose the ticket till #278 . cluster.get_os_identifiers() is introduced there and can be used.

Reply:
okay
Used same logic which is used for ticket #278 in check_paas to check the os versions now